cognite/powerops/custom_modules/power_model_v1/data_models/containers/ShopModel.container.yaml

```diff
@@ -41,23 +41,7 @@ properties:
     type:
       list: false
       type: file
-    nullable: false
+    nullable: true
     autoIncrement: false
     name: model
     description: The shop model file to use as template before mapping
-  cogShopFilesConfig:
-    type:
-      list: true
-      type: json
-    nullable: true
-    autoIncrement: false
-    name: cogShopFilesConfig
-    description: Configuration for in what order to load the various files into pyshop
```

Comment:
I thought we needed this - is it replaced by a new type, or don't we need it? As far as I know, this is something else than the shopFiles on ShopCase (which point to specific files) - cogShopFilesConfig may contain information about how to find/pick the right file (like the water value cut files, for which there are several new files every week).

Reply:
It's been switched to a `multi_edge_connection` of ShopFile so the property definition has been moved to the view. I synced with Julia about it, and the contents of the JSON needed to be the same as what is defined in the ShopFile object, so I just made `cogShopFilesConfig a list of ShopFile` instead.

Reply:
Also, this change is detailed in the CHANGELOG, so it would be good if you referred to that as well to understand the implication of the changes.

Reply:
So we will use a ShopFile without fileReference for this?

Reply:
Sorry this is my fault. I told Nina to change it to use a list of ShopFiles as input to the preprocessor as well, but since we do not know the exact `water_value_cut_file` to use before doing a lookup in CDF based on the latest ingested file, we should keep the list of json for the cogShopFilesConfig. I was looking at the file we had in powerops-staging that references the cut file directly, but this is not the case in customer environments. Example:


``` 
 - label: "water_value_cut_file"
    pick: "latest_before"  # pick the file that is closest before the start time to the triggering event
    sort_by:
      metadata_key: "update_datetime"
      file_attribute:
    external_id_prefix: "SHOP_{WATERCOURSE}_water_value_cut_file"
    file_type: "ascii"
```

Reply:
Resolved with @Juliamg by adding an optional `external_id_prefix` field to ShopFile